packages/components/pie-link/src/link.scss

```diff
@@ -18,6 +18,18 @@
     text-decoration: var(--link-text-decoration);
     cursor: pointer;
 
+    &[tag='a'] {
+        /* Same as default styles */
+    }
+
+    &[tag='button'] {
+        outline: none;
```

Comment:
I can't seem to find any focus styles in storybook. Could this be removing them?

Reply:
focus, active, and hover states are not done yet as designers are still looking into them 💯 
DSW-1057